label_studio/users/admin.py

```diff
@@ -61,7 +61,99 @@ def __init__(self, *args, **kwargs):
         self.list_display = ('id', 'name', 'project', 'status', 'created_at', 'updated_at', 'meta')
         self.list_filter = ('name', 'status')
         self.search_fields = ('name', 'project__id')
-        self.ordering = ('id',)
+        self.ordering = ('-id',)
+        self.actions = ['run_scheduled_migrations']
+
+    def run_scheduled_migrations(self, request, queryset):
+        """Run selected scheduled migrations manually.
+
+        Expects AsyncMigrationStatus.name in one of forms:
+        - "<app>:<migration_module>"
+        - "<app>.migrations.<migration_module>"
+        - Full dotted path "label_studio.<app>.migrations.<migration_module>"
+
+        Does not scan all apps. If app is not provided, marks error.
+        """
+        import importlib
+
+        from core.migration_helpers import execute_sql_job
+        from core.redis import start_job_async_or_sync
+
+        executed_count = 0
+        skipped_count = 0
+        error_count = 0
+
+        def resolve_import_path(name: str) -> tuple[str | None, str | None]:
+            s = (name or '').strip()
+            if not s:
+                return None, 'Empty migration import path'
+            if '.migrations.' not in s:
+                return None, 'Migration import path must include ".migrations." (use full dotted path)'
+            return s, None
+
+        for migration in queryset:
+            if migration.status != migration.STATUS_SCHEDULED:
+                skipped_count += 1
+                continue
+
+            import_path, err = resolve_import_path(migration.name)
+            if err:
+                meta = migration.meta or {}
+                meta['error'] = err
+                migration.meta = meta
```

Comment:
this piece of code is duplicated 4 times, let's move it to helper function? 
This way we will  increase coverage by tests automatically:
https://app.codecov.io/gh/HumanSignal/label-studio/pull/8767?src=pr&el=tree&filepath=label_studio%2Fusers%2Fadmin.py&utm_medium=referral&utm_source=github&utm_content=checks&utm_campaign=pr+comments&utm_term=HumanSignal#diff-bGFiZWxfc3R1ZGlvL3VzZXJzL2FkbWluLnB5